docs/src/tricks/gpu.jl

```diff
@@ -0,0 +1,78 @@
+# # Using DFTK on GPUs
+#
+# In this example we will look how DFTK can be used on
+# Graphics Processing Units.
+# In its current state runs based on Nvidia GPUs
+# using the [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl) Julia
+# package are better supported and there are considerably less rough
+# edges.
+#
+# !!! info "GPU parallelism not supported everywhere"
+#     GPU support is still a relatively new feature in DFTK.
+#     While basic SCF computations and e.g. forces are supported,
+#     this is not yet the case for all parts of the code.
+#     In most cases there is no intrinsic limitation and typically it only takes
+#     minor code modification to make it work on GPUs.
+#     If you require GPU support in one of our routines, where this is not
+#     yet supported, feel free to open an issue on github or otherwise get in touch.
+#
+
+using AtomsBuilder
+using DFTK
+using PseudoPotentialData
+
+# **Model setup.** First step is to setup a [`Model`](@ref) in DFTK.
+# This proceeds exactly as in the standard CPU case
+# (see also our [Tutorial](@ref)).
+
+silicon = bulk(:Si)
+
+model  = model_DFT(silicon;
+                   functionals=PBE(),
+                   pseudopotentials=PseudoFamily("dojo.nc.sr.pbe.v0_4_1.standard.upf"))
+nothing  # hide
+
+# Next is the selection of the computational architecture.
+# This effectively makes the choice, whether the computation will be run
+# on the CPU or on a GPU.
+#
+# **Nvidia GPUs.**
+# Supported via [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl).
+# Right now `Libxc` only supports CUDA 11,
```

Comment:
I have a PR to Yggdrasil for Libxc 7, and full CUDA support. It's been dormant for a while, I'll try to get it merged.